src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -796,6 +783,30 @@ public void ErrorListInfoBarController_InfoBar_ClickButton_MoreThanOnce()
             bindCommandExecuted.Should().Be(1, "Expecting the command to be executed only once");
         }
 
+        [TestMethod]
```

Comment:
Added critical/non-critical exception tests for the one event handler that could be easily tested.